config/routes.rb

```diff
@@ -5,6 +5,11 @@
   resources :users
   resources :apartments
   resources :orders
+  resources :contacts
+
+  namespace :admin do
```

Comment:
if admin can't edit users messages, so you don't need routes for this. Please specify only need routes